contracts/oracle/src/lib.rs

```diff
@@ -314,51 +323,51 @@ mod tests {
         let (_, c_str, sig) = mk_claim_sign(start() / SECOND, "0x1a", &k);
         ctx.signer_account_id = acc_u1();
         testing_env!(ctx.clone());
-        match ctr.sbt_mint(c_str.clone(), sig.clone()) {
+        match ctr.sbt_mint(c_str.clone(), sig.clone(), None) {
             Err(CtrError::BadRequest(s)) => assert_eq!(s, "claimer is not the transaction signer"),
-            resp @ _ => panic!("expected BadRequest, got: {:?}", resp),
+            _resp @ _ => panic!("expected BadRequest"),
         }
 
         // fail: claim_ttl passed
         ctx.signer_account_id = signer.clone();
         ctx.block_timestamp = start() + CLAIM_TTL * SECOND;
         testing_env!(ctx.clone());
-        match ctr.sbt_mint(c_str.clone(), sig.clone()) {
+        match ctr.sbt_mint(c_str.clone(), sig.clone(), None) {
             Err(CtrError::BadRequest(s)) => {
                 assert_eq!("claim expired", s, "wrong BadRequest: {}", s)
             }
-            resp @ _ => panic!("expected BadRequest, got: {:?}", resp),
+            _resp @ _ => panic!("expected BadRequest"),
```

Comment:
why changing this?

Reply:
but here we know it's an Error , not a Promise.

Reply:
That was the problem: ``near_sdk::Promise` doesn't implement `Debug`
the trait `Debug` is not implemented for `near_sdk::Promise`` . Fixed it

Reply:
how did you fix it?

Reply:
Pushed the changes